### DISCUSSION TITLE
Support SQLite databases

### DISCUSSION THREAD

**Discussion by Gui11aum3** [WebDB-App/app#18] [Ideas]
It's not easy to find a simple webUI to work with SQLite stored data. 
Any plans on supporting SQLite format?

**Comment by WebDB-App** [comment]
It's in the TODO list
I don't know now, which one will be done first between SQL Server and SQLite
Maybe I will just let upvotes number decide for me

**Comment by am-py** [comment]
There is this tool to query sqlite in the browser :
https://sqliteonline.com
I'd like to see something similar with more features like pasting a table from excel
